src/Quote/StrictQuoteLine.php

```diff
@@ -2,6 +2,10 @@
 namespace ClickHouseDB\Quote;
 
 use ClickHouseDB\Exception\QueryException;
+use function array_map;
+use function is_string;
+use function preg_replace;
```

Comment:
Do you think this is clear code ?

Reply:
Of course. Either you can use `\array_map` in your code or import it. Importing it is cleaner so your actual code is not polluted with `\`s

Reply:
@isublimity what do you think about that ?

Reply:
@simPod you are not right 
<img width="963" alt="2018-08-22 23 34 21" src="https://user-images.githubusercontent.com/4361999/44489369-0673e880-a664-11e8-8f92-c9868d22b4fa.png">

Reply:
I'm. It is part of the code style to reference it via use statement instead of prepending \. What is the issue you see there?

Reply:
@simPod you said "you can use \array_map in your code or import it.", I show you errors about use `\` in function name

Reply:
I meant that it both have the same effect but only one is compliant with doctrine cs and that's by importing it.

Reply:
@simPod - please write me if need merge, i'm lost discussion( 
sorry(

Reply:
@isublimity I think this PR is ready. I have leveraged the code used for encoding strings https://github.com/smi2/phpClickHouse/pull/87/files#diff-c29b9c40206dcbba3b525b3b3d284effL82 and reapplied it for arrays as well.

The test written by @2ur1st doesn't pass before this change so I suppose it's fixed now.